spyder/widgets/explorer.py

```diff
@@ -42,6 +44,26 @@
 except:
     nbexporter = None    # analysis:ignore
 
+def open_file_in_external_explorer(filename):
+    if sys.platform == "darwin":
+        subprocess.call(["open", "-R", filename])
+    else:
+        filename=os.path.dirname(filename)
+        if sys.platform == "win32":

```

Comment:
Please change this to

    if os.name == 'nt':

That's how we check for Windows in all our codebase